packages/php-datatypes/src/Definition/Bigquery.php

```diff
@@ -310,4 +329,22 @@ private function validateNumericScaleAndPrecision(string $length, int $decimalLe
 
         return false;
     }
+
+    /**
+     * @param null|int|string $length
+     */
+    //phpcs:ignore SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint
+    private function validateRepeatedType($length): bool
+    {
+        // array and struct has not default "length" it fails on BQ so we not validate in this lib
+//        if ($this->isEmpty($length)) {
+//            return false;
+//        }
+
+        if (is_numeric($length)) {
+            return false;
+        }
+
+        return true;
+    }
```

Comment:
není to celé zbytečné když to teda nevalidujeme. Pokud ne tak mě chybý test kde failne ta length

Reply:
Jo mas pravdu, chcel som to aspon nejak zakladne validovat ale je to asi zbytocne